CHECKLIST

```diff
@@ -24,6 +24,8 @@ The topics can be an empty list, and the difficulty level is from 1 to 10, and c
 ]
 ```
 
+The order of the exercises is decided by the order in this "exercises" array. Try to make them roughly increasing in difficulty, but don't worry too much—we can always adjust the order later, as we get feedback from people doing the exercises.
```

Comment:
So what is the plan with the `difficulty` ranking? Are we going to use this only as an indication to contributors on where to place the exercise in the list of exercises or are we going to use those at some point to sort the exercises programmatically?